content/docs/tasks/security/health-check/index.md

```diff
@@ -115,11 +130,16 @@ continuously failed health checks.
 
 ## Cleanup
 
-*   To disable health checking on Citadel, deploy Citadel with health checking disabled by setting the Helm argument `security.healthCheckEnabled` as `false`.
+*   To disable health checking on Citadel:
+
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+    $ kubectl delete svc istio-citadel -n istio-system
```

Comment:
We don't need to delete the service. The service is enabled by default now.

Reply:
Done: no deleting the service.